src/Chapter13/Listing13.18.UsingVarianceForDelegates.cs

```diff
@@ -17,9 +17,10 @@ public static void Main()
 
             Action<string> narrowAction = broadAction;
 
+            // TODO: Update listing in Manuscript
             // Covariance
             Func<string> narrowFunction =
-                () => Console.ReadLine();
+                () => 8654091021.ToString();
```

Comment:
If we are just going to return a hard coded string I would just use a string literal. 
```suggestion
                () => "8654091021";
```